### DISCUSSION TITLE
Notify bubble, postitioned on a component, not window corner

### DISCUSSION THREAD

**Discussion by andrisi** [quasarframework/quasar#14852] [Ideas / Proposals]
I'd like to to use the Notify API, with actions, but instead of positioning in various corners, I'd position it on a component. So the user clicks the Favourite button, and then a few options appear. Would be nice to have a tip pointing to the element, like on the image below, and then the words would be links and thus actions. It's sort of a tooltip, but the Tooltip component does not have actions.

![image](https://user-images.githubusercontent.com/1255809/200627425-ee062e0e-1ca3-41ab-a665-f24ed03fe3eb.png)

**Comment by suleimanas** [comment]
you can use QMenu, like:
```
<q-btn color="primary" label="keress">
        <q-menu>
          //anything          
          <q-card>
            <q-card-section>
              abcd
            </q-card-section>
          </q-card>
         </q-menu>
      </q-btn>
```
https://quasar.dev/vue-components/menu

**Reply by andrisi** [reply]
Thanks @suleimanas I'll probably use this as a fallback, but this lacks the autohide feature of the Notity API, which I need and is attached to the button visually, which I don't like. It really needs to be that floating thing as the Notify bubble.
